app/controllers/users_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def show
     @user = User.includes(:posts).find_by(id: params[:id]) || 'There is no user'
+    @posts = @user.posts.limit(3).page(params[:page])
   end
```

Comment:
- [ ] As stated in the requirements, let's kindly resolve the N+1 problem when fetching **All** posts and their comments for a user not a single post. The problem should probably be resolved in the posts#index action

Reply:
Done